fix: Ignore NRF URI containing a specific IP on registration

ghislainbourgeois opened the pull request:
This ignores the NRF URI received from the NRF on registration, as the NRF will return its IP, which is subject to change. This is similar to what is done on the AMF.

gab-arrobo commented:
ok to test

gab-arrobo commented on factory/config.example.yaml:
How is this file used? I do not see being called from anywhere. If this file is required, the "License verification" fails because this file does not have License/Copyright header.

ghislainbourgeois replied:
It is used by the test I added, it needs to read a configuration file to initialize before the test. I added the license and copyright.

gab-arrobo commented:
@ghislainbourgeois please rebase your PR. Thanks!

ghislainbourgeois commented:
> @ghislainbourgeois please rebase your PR. Thanks!

@gab-arrobo I just rebased and added license and copyright to the file that was missing it. Thanks!

gab-arrobo commented on factory/config.example.yaml:
```diff
@@ -0,0 +1,22 @@
+  description: UDR initial local configuration
+
+configuration:
+  sbi: # Service-based interface information
```
As per its abbreviation in 23.501:
```suggestion
  sbi: # Service Based Interface
```

gab-arrobo commented:
ok to test

gab-arrobo reviewed (approved):
+1
